composer.json

```diff
@@ -39,7 +39,7 @@
         }
     },
     "require": {
-        "php": "~8.1.0",
+        "php": "^8.1 || ^8.2",
```

Comment:
`~8.1.0 || ~8.2.0`

Reply:
In this case, using [tilde](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-) or [caret](https://getcomposer.org/doc/articles/versions.md#caret-version-range-) would be the same, right?

If I understood correctly both use [semantic version](https://semver.org/) but the tilde will always allow an increment of the last digit (except major) while caret always allows patch.